src/services/BavService.ts

```diff
@@ -63,6 +63,31 @@ export class BavService {
 		}
 	}
 
+	async getPersonIdentityBySessionId(sessionId: string, tableName: string = this.tableName): Promise<PersonIdentityItem | undefined> {
+		this.logger.debug("Table name " + tableName);
```

Comment:
Do you think it would be helpful to say what we are doing with this table name? Something like `Getting person identity from table ${tableName}`

Reply:
Done